src/com/palmergames/bukkit/towny/object/Town.java

```diff
@@ -1483,4 +1485,13 @@ public boolean pay(double amount, String reason) throws EconomyException {
 	public boolean collect(double amount, String reason) throws EconomyException {
 		return getAccount().collect(amount, reason);
 	}
+	
+	/**
+	 * @deprecated As of 0.96.0.1, please use {@link Town#getHomeblockWorld()} instead.
+	 */
+	@Deprecated
+	public TownyWorld getWorld() {
+		return getHomeblockWorld();
+	}
+	
```

Comment:
Is there a particular reason for this deprecation, It seems more intuitive to use Town.getWorld() as opposed to Town.getHomeBlockWorld(), they also essentially do the same thing.

Reply:
Because the code that uses it is specifically asking for the HomeBlockWorld.